### DISCUSSION TITLE
a/nginx-proxy-poe-ai

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#7] [Announcements]
# a/nginx-proxy-poe-ai

https://blog.yasking.org/a/nginx-proxy-poe-ai.html

**Comment by Tsenghan** [comment]
你好，我试用了你的方法，显示502错误，请问你知道原因吗

**Reply by nianganyu** [reply]
我也是一样的问题

**Reply by sincerefly** [reply]
感谢反馈，Nginx 配置问题，已修复，详见「2024-04-25 更新」的补充。

**Comment by smctest6** [comment]
請問用這個方法還 work 嗎? Login 時, 我遇到以下的 response


Method 1 (phone number):

{
"data": {
"sendVerificationCode": {
"errorMessage": "Your device or network may be sending automated queries. We can't process your request now.",
"status": "automated_request_detected"
}
},
"extensions": {
"is_final": true
}
}


Your device or network may be sending automated queries. We can't process your request now.



Method 2 (Email):

Sign in with Google
Access blocked: This app’s request is invalid


You can’t sign in because this app sent an invalid request. You can try again later, or contact the developer about this issue. Learn more about this error
If you are a developer of this app, see error details.
Error 400: redirect_uri_mismatch

**Reply by sincerefly** [reply]
感觉是你的账号或网络被 Ban 了
